src/client/post/EditPost/EditPost.js

```diff
@@ -240,10 +257,10 @@ class EditPost extends Component {
     const currDraft = this.props.draftPosts.find(d => d.draftId === this.props.draftId);
     const oldMetadata = currDraft && currDraft.jsonMetadata;
     const waivioData = {
-      wobjects: linkedObjects.map(obj => ({
+      wobjects: postObjects.map(obj => ({
         objectName: obj.name,
-        author_permlink: obj.id,
-        percent: objPercentage[obj.id].percent,
+        author_permlink: obj.id || obj.author_permlink,
+        percent: objPercentage[obj.id || obj.author_permlink].percent,
```

Comment:
You could use only 'id' if apply 'getClientWObj' to all linked objects

Reply:
++